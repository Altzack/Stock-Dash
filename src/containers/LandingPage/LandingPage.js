import React from 'react';
import NewsList from '../NewsList/NewsList';
import Graph from '../Graph/Graph';
import WatchList from '../WatchList/WatchList';

function LandingPage() {
  return (
    <div style={{ color: '#000', backgroundColor: 'rgb(27, 29, 30)' }}>
      <div>
        <Graph />
      </div>
      <div>
        <WatchList />
      </div>
      <div>
        <NewsList />
      </div>
    </div>
  );
}

export default LandingPage;
