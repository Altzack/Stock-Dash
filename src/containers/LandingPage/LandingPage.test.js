import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LandingPage from '../About/About';

it('renders a landingpage by default', () => {
  const wrapper = shallow(<LandingPage />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
