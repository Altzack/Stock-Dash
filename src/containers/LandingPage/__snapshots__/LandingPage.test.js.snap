// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a landingpage by default 1`] = `
<About__PageContainer
  mobile={false}
>
  <About__Title>
    About Stock-Dash
  </About__Title>
  <br />
  <About__ContentContainer>
    Stock-Dash was built as my second full stack capstone for Bloc.
    <br />
    <br />
    The idea for Stock-Dash came from my interest in trading stocks. The dashboard allows you to get the latest news and pricing for a symbol, as well as see the past month's price data visualized. With the watchlist feature you can easily navigate to your top 5 and keep an eye on your favorite tickers.
    <br />
    <br />
  </About__ContentContainer>
</About__PageContainer>
`;
